### DISCUSSION TITLE
Display output of the node below

### DISCUSSION THREAD

**Discussion by theaspirational** [newcat/baklavajs#365] [Q&A]
is it possible to add the output of the node's compute function(e.g. some logs or other custom output) at the bottom of the node itself? 

something like this

![CleanShot 2024-01-16 at 20 02 09](https://github.com/newcat/baklavajs/assets/48733736/380b0b41-2dbd-461a-8fe6-4442ebc71dae)
![CleanShot 2024-01-16 at 20 05 13](https://github.com/newcat/baklavajs/assets/48733736/88d98512-3ceb-4942-8c5a-b8e26f0dca3e)

**Comment by newcat** [comment]
Currently, I think the only way is to override the node component with a custom component ([docs](https://v2.baklava.tech/visual-editor/customization.html#custom-components)). Unfortunately, there is no easy way to just add stuff to a node, so you'd have to reimplement the whole `Node` component and add the logs display at the bottom. Then you can, for example, add a hidden `logs` output to every node. During calculation, you could then assign whatever value you want to that output and display the value at the end using your custom node component.
